output/adc2vtk.pl

```diff
@@ -477,6 +478,25 @@
 # not needed #printf XDMFFLUXBOUNDARY "          <Geometry Type=\"None\"/>\n";
 # not needed #printf XDMFFLUXBOUNDARY "             <Topology Dimensions=\"0\" Type=\"NoTopology\"/>\n";
 #
+#
+# write out the flux-specified boundary tables as ADCIRC fort.14 mesh geometry
+# to show boundary height ; start with a node table file and an element
+# table file, build them up separately, accumulating the number of nodes and
+# the number of elements and then concatenate them into an adcirc mesh
+# file with the number of elements and nodes at the top
+my $adcFluxBoundaryGeometryNodeFileName = $meshfile . "_fluxBoundaryGeometry.nod";
+unless (open(ADCNODFLUXBOUNDARY,">$adcFluxBoundaryGeometryNodeFileName")) {
+   stderrMessage("ERROR","Failed to open '$adcFluxBoundaryGeometryNodeFileName' for writing: $!.");
+   die;
+}
+my $adcNodeID = 1;
+my $adcFluxBoundaryGeometryElementFileName = $meshfile . "_fluxBoundaryGeometry.ele";
+unless (open(ADCELEFLUXBOUNDARY,">$adcFluxBoundaryGeometryElementFileName")) {
```

Comment:
you're using the 2-argument `open`, consider making this more consistent with:

```
unless (not open (my $ADCELEFLUXBOUNDARY, ">", $adcFluxBoundaryGeometryElementFileName")) {
```
You'll have to make sure `ADCELEFLUXBOUNDARY` is used below as `$ADCELEFLUXBOUNDARY`,